src/plugins/links/public/components/editor/links_editor.tsx

```diff
@@ -67,16 +66,16 @@ const LinksEditor = ({
   onClose,
   initialLinks,
   initialLayout,
-  parentDashboard,
+  parentDashboardId,
   isByReference,
   flyoutId,
 }: {
-  onSaveToLibrary: (newLinks: Link[], newLayout: LinksLayoutType) => Promise<void>;
-  onAddToDashboard: (newLinks: Link[], newLayout: LinksLayoutType) => void;
+  onSaveToLibrary: (newLinks: ResolvedLink[], newLayout: LinksLayoutType) => Promise<void>;
+  onAddToDashboard: (newLinks: ResolvedLink[], newLayout: LinksLayoutType) => void;
   onClose: () => void;
-  initialLinks?: Link[];
+  initialLinks?: ResolvedLink[];
   initialLayout?: LinksLayoutType;
```

Comment:
The `initialLayout` isn't being passed properly:


https://github.com/elastic/kibana/assets/8698078/230f59f3-4ee7-4f6f-ae42-aabd6804f105

Reply:
🤦 https://github.com/elastic/kibana/commit/3305d783440519a28656f4abe133684186827ccf